nservicebus/messaging/third-party-integration.md

```diff
@@ -4,31 +4,31 @@ summary: Receiving messages from external systems
 
-Depending on the serializer used by the receiving endpoint, that information might also be provided in other ways. 
+The sender can specify a message type with the [NServiceBus.EnclosedMessageTypes header](/nservicebus/messaging/headers.md#serialization-headers-nservicebus-enclosedmessagetypes).
 
-Embedding type info in the message body is currently supported by the following serializers
+Some serialiazers can infer the message type from information embedded in the message body. 
 
  * [Xml](/nservicebus/serialization/xml.md) 
  * [Json](/nservicebus/serialization/json.md)
  * [Newtonsoft](/nservicebus/serialization/newtonsoft.md)
 
-The [RabbitMQ](/samples/rabbitmq/native-integration/), [SQL](/samples/sqltransport/native-integration/), and [Azure Service Bus](/samples/azure/native-integration-asb/) native integration samples demonstrates this.
+The [RabbitMQ](/samples/rabbitmq/native-integration/), [SQL](/samples/sqltransport/native-integration/), and [Azure Service Bus](/samples/azure/native-integration-asb/) native integration samples demonstrate inferring message type from .
 
 
-## Additional information
+## Visualization
 
 To visualize messages from third party systems correctly within ServiceInsight, additional headers are necessary
 
- * [NServiceBus.ConversationId](/nservicebus/messaging/headers.md#messaging-interaction-headers-nservicebus-conversationid) If not provided, then the external message won't be included in the conversation it started. ConversationId will be generated though for all the following messages sent using NServiceBus.
- * [NServiceBus.OriginatingEndpoint](/nservicebus/messaging/headers.md#diagnostic-and-informational-headers-nservicebus-originatingendpoint) Name of the third party endpoint sending the message, for e.g. *BizTalk.ProcessOrder*
- * [NServiceBus.OriginatingMachine](/nservicebus/messaging/headers.md#diagnostic-and-informational-headers-nservicebus-originatingmachine) Server where the third party endpoint is located, for e.g. *BizTalkServer*
+ * [NServiceBus.ConversationId](/nservicebus/messaging/headers.md#messaging-interaction-headers-nservicebus-conversationid) - is used to show related messages in the message flow and sequence views. If not provided, then the external message won't be included in the conversation it started. ConversationId will be generated though for all the following messages sent using NServiceBus.
+ * [NServiceBus.OriginatingEndpoint](/nservicebus/messaging/headers.md#diagnostic-and-informational-headers-nservicebus-originatingendpoint) - is used to show the logical endpoint that sent the message in all views. This should be the name of the third party endpoint sending the message. e.g. *BizTalk.ProcessOrder*
+ * [NServiceBus.OriginatingMachine](/nservicebus/messaging/headers.md#diagnostic-and-informational-headers-nservicebus-originatingmachine) - Server where the third party endpoint is located. e.g. *BizTalkServer*
```

Comment:
Not sure if this is actually used by ServiceInsight. @Particular/serviceinsight-maintainers does anyone know off the top of their head?

Reply:
@mikeminutillo AFAICT it's not used explicitly. It just shows in the Headers view (obviously).